### PR TITLE
Update sensors.mitemp_bt configuration variables

### PR DESCRIPTION
**Pull request by Mryck**
**Description:**

Update sensors.mitemp_bt configuration variables #6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.mitemp_bt.markdown [hunk 1]
```diff
@@ -60,18 +60,49 @@ sensor:
...
+{% configuration %}
+mac:
+  description: The MAC address of your sensor.
+  required: true
+  type: string
+monitored_conditions:
+  description: The parameters that should be monitored.
+  required: false
+  default: [temperature, humidity, battery]
+  type: list
```
Comment (frenck) [inline source/_components/sensor.mitemp_bt.markdown:72]: Please define the `keys` section.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Overall it looks good @Mryck, just one small adjustment. Could you take a look? 👍

**Comment by Mryck** [comment]
Thanks @frenck it should be fine now.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the update @Mryck!

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**